src/esp/physics/bullet/objectWrappers/ManagedBulletRigidObject.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_PHYSICS_MANAGEDBULLETRIGIDOBJECT_H_
+#define ESP_PHYSICS_MANAGEDBULLETRIGIDOBJECT_H_
+
+#include "esp/physics/bullet/BulletRigidObject.h"
+#include "esp/physics/objectWrappers/ManagedRigidObject.h"
+
+namespace esp {
+namespace physics {
+
+/**
+ * @brief Class describing wrapper for RigidObject constructions.
+ * Provides bindings for all RigidObject-specific functionality.
+ */
+class ManagedBulletRigidObject : public esp::physics::ManagedRigidObject {
+ public:
+  ManagedBulletRigidObject() : ManagedRigidObject("ManagedBulletRigidObject") {}
+
+  double getMargin() const {
+    if (auto sp = this->getBulletObjectReference()) {
+      return sp->getMargin();
+    } else {
+      return 0.0;
+    }
+  }  // getMargin
+
+  void setMargin(const double margin) {
+    if (auto sp = this->getBulletObjectReference()) {
+      sp->setMargin(margin);
+    }
+  }  // setMass
+
+  Magnum::Range3D getCollisionShapeAabb() {
+    if (auto sp = this->getBulletObjectReference()) {
+      return sp->getCollisionShapeAabb();
+    } else {
```

Comment:
Avoid else return. just do return underneath. Working on a clang-tidy rule to fix this since I see It a lot in the code base.

Reply:
Due to the nature of the wrappers, default results needs to be returned if the underlying object being wrapped is invalid (these may not always be the default values for the empty shared pointer).

Reply:
That's a pretty big refactor (all the wrappers). I'll fix it for ManagedBulletRigidObject in this PR and put another one up that addresses it elsewhere.

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html

Reply:
Actually, nevermind, I fixed all of them, easy enough.